### PR TITLE
Update new registration workflow

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://trello.com/c/j3zS2qqI/689-online-update-docs-on-unified-registration-application

Preview build: http://file.rdu.redhat.com/~ahardin/10052017/unified-registration/getting_started/choose_a_plan.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@tiwillia PTAL at my first pass of updates. Thanks!

File: getting_started/choose_a_plan.adoc [hunk 1]
```diff
@@ -25,10 +25,11 @@ By Example].
 [[customize-your-plan-with-add-ons]]
 == Customize Your Plan with Add-ons
 
-If you choose the {product-title} Pro plan, you can customize your plan with
-add-ons.
+You can customize your plan with add-ons.
 
-. In the *My Account* page, click *Change* by the *Current Plan* heading.
+. In the *Active Subscriptions* page, click the *Manage Subscription* link.
```
Comment (ahardin-rh) [inline getting_started/choose_a_plan.adoc]: Once Jake updates the UI for this page, I can add a screenshot here as well.

**Comment by ahardin-rh** [comment]
@tiwillia Updated with latest screenshots following our discussion. Thanks!

cc @abhgupta

**Comment by tiwillia** [comment]
Looks fantastic  💯 👍 🎉 , thanks @ahardin-rh

**Review by tiwillia — APPROVED** [review]

**Pull request merged**